package.json

```diff
@@ -22,12 +22,13 @@
     "@types/react": "^18.0.9",
     "@types/react-router-dom": "^5.3.3",
     "@types/rx": "^4.1.1",
-    "firebase": "^9.8.1",
+    "firebase": "^9.15.0",
     "firebase-tools": "11.x",
     "gulp": "^4.0.2",
     "jest": "^23.6.0",
     "microbundle": "^0.15.0",
     "prettier": "^2.8.3",
+    "prettier-plugin-organize-imports": "^3.2.2",
```

Comment:
This prettier package is an aesthetic choice, it shouldn't be included in this PR, as there's tonnes of import changes that have no effect on the behaviour of the system.

Mixing linting & functional changes in the same PR's makes the git history confusing and hard to see exactly what changes something.

I'll allow this, but next time, please restrict PR's to the smallest change necessary to make review easier. Linting PR's on the other hand, can be huge as they're _not functional changes_.

Reply:
@benwinding I reverted all the `prettier` and `tslint` changes, but this one slipped in somehow.

Let me know if you want me to revert this back.

Reply:
Yeah that would be great actually 🙏 Since this is a major firebase upgrade 👍

Reply:
Or just make another PR with the `prettier-plugin-organize-imports` linting changes, that can get merged in before (so these changes will be reduced)